README.md

```diff
@@ -320,98 +805,368 @@ And configure your remote endpoint:
+_Reminder: These settings go into your `package.json` or [Hops configuration file](#settings)._
 
-## FAQ
+| Name                             | Type     | Default                                        | Description                                                                                        |
+| -------------------------------- | -------- | ---------------------------------------------- | -------------------------------------------------------------------------------------------------- |
+| `aws.region`\*                   | `String` | `us-east-1`                                    | The AWS Region in which the resources should be created                                            |
+| `aws.uniqueName`                 | `String` | `hops-lambda-$name`                            | A unique name that is used to identify the AWS CloudFormation Stack and S3 bucket.                 |
+| `aws.memorySize`                 | `Number` | `128`                                          | The memory allocated to your Lambda function                                                       |
+| `aws.stageName`                  | `String` | `prod`                                         | The name of your API Gateway stage                                                                 |
+| `aws.domainName`                 | `String` | `''`                                           | A custom domain name                                                                               |
+| `aws.certificateArn`             | `String` | `''`                                           | If a custom domain is used, this option needs to specify the ARN of a valid SSL certificate in ACM |
+| `aws.cloudformationTemplateFile` | `String` | `node_modules/hops-lambda/cloudformation.yaml` | Path to a custom CloudFormation template                                                           |
 
-### When is Hops the framework for you?
+_\* If no region is configured via the preset config, `hops-lambda` will try to read `AWS_REGION` and `AWS_DEFAULT_REGION` from your environment first before defaulting to `us-east-1`._
 
-When you want to share a consistent setting over more than one project. Either now or in the future. This applies to companies with more than one development team or when you constantly set up projects that should be consistent, e.g. training projects, tutorials, consultant work, etc.
+For more details and more advanced use-cases, head over to the [full readme of `hops-lambda`](https://github.com/xing/hops/tree/master/packages/lambda).
 
-Even without striving for consistency across multiple projects you may want to use Hops for a (single) new project when you do not want to write/copy/maintain your own webpack configuration. We feel your pain and have done that for you. However, we try hard to give you all the customizability you may ever need.
+## Advanced configuration and extension
 
-### When is Hops **not** the framework for you?
+### Debugging
```

Comment:
This section relies on using yarn while we stick to npm in the rest of the document.